foursight_core/react/api/react_routes.py

```diff
@@ -108,42 +109,59 @@ def reactapi_route_user_get_or_patch_or_delete(env: str, uuid: str) -> Response:
         if the methods are different; rather you must bundle them together and
         distinguish between which method is used programmatically as we do here.
         """
+        request = app.current_request.to_dict()
```

Comment:
Well, React has little/nothing to do with this code here (maybe missing your point WRT to that comment), but you're probably right, best to name this variable request_dict, will make this change.

Reply:
I just didn't want to be trying to override some other naming convention that was useful in some other place in the data flow. But yeah, if you can take the renaming I think it will help.